src/ajitSJune20/TestReturnTypeStaticNonStatic.java

```diff
@@ -0,0 +1,49 @@
+package ajitSJune20;
+
+public class TestReturnTypeStaticNonStatic {
+
+	double sum(double var1, double var2, double var3) {
+
+		System.out.println(
+				"Sum of three numbers    (" + var1 + " , " + var2 + " , " + var3 + "): " + (var1 + var2 + var3));
+		return (var1 + var2 + var3);
+	}
+
+	double average(double var1, double var2, double var3) {
+
+		System.out.println("\nAverage of three numbers(" + var1 + " , " + var2 + " , " + var3 + "): " + (var1 + var2 + var3) / 3);
```

Comment:
If you perform the same operation multiple time then store in local variable, and use sum() method.

Reply:
Incorporated review Comments.